src/applications/vaos/tests/appointment-list/components/CommunityCareAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -299,4 +301,57 @@ describe('VAOS <CommunityCareAppointmentDetailsPage>', () => {
       }),
     ).to.be.ok;
   });
+
+  it('should show cc appointment from vista when directly opening page', async () => {
+    const url = '/cc/8a4885896a22f88f016a2cb7f5de0062';
+
+    const appointment = getVAAppointmentMock();
+    appointment.id = '8a4885896a22f88f016a2cb7f5de0062';
+    appointment.attributes = {
+      ...appointment.attributes,
+      clinicId: '308',
+      clinicFriendlyName: 'COMMUNITY CARE',
+      facilityId: '983',
+      sta6aid: '983GC',
+      communityCare: true,
+      vdsAppointments: [
+        {
+          bookingNote: '',
+          appointmentLength: '60',
+          appointmentTime: '2021-12-07T16:00:00Z',
+          clinic: {
+            name: 'CHY OPT VAR1',
+            askForCheckIn: false,
+            facilityCode: '983',
+          },
+          type: 'REGULAR',
+          currentStatus: 'FUTURE',
+        },
+      ],
+      vvsAppointments: [],
+    };
+
+    mockSingleVistaCommunityCareAppointmentFetch({
+      appointment,
+    });
+
+    const screen = renderWithStoreAndRouter(<AppointmentList />, {
+      initialState,
+      path: url,
+    });
+
+    expect(
+      await screen.findByRole('heading', {
+        level: 1,
+        name: new RegExp(
+          moment()
+            .tz('America/Denver')
+            .format('dddd, MMMM D, YYYY'),
+          'i',
+        ),
+      }),
+    ).to.be.ok;
+
+    expect(screen.getByText(/Community care/)).to.be.ok;
+  });
```

Comment:
I'm sure I'm missing something but is this just a missing test or is it related to the modification to fetch VA appointments when no CC appointments are found?

Reply:
It's related. It uses a different mock, `mockSingleVistaCommunityCareAppointmentFetch`, which is set up to mock the scenario where we try the CC service first, then fall back to MAS.